registry/auth.go

```diff
@@ -180,6 +180,8 @@ func loginV1(authConfig *cliconfig.AuthConfig, registryEndpoint *Endpoint) (stri
 				}
 				// *TODO: Use registry configuration to determine what this says, if anything?
 				return "", fmt.Errorf("Login: Account is not Active. Please see the documentation of the registry %s for instructions how to activate it.", serverAddress)
+			} else if resp.StatusCode == 500 { // Issue #14326
+				return "", fmt.Errorf("Internal Server Error")
```

Comment:
I could, but I am not sure what else to add. The `body` is a huge html dump from the `index` service. I could update the Error message to something more appropriate like `index.docker.io returned a 500 while trying to authenticate`, is that what you're looking for?